package.json

```diff
@@ -68,5 +68,6 @@
 	},
 	"svelte": "./dist/index.js",
 	"types": "./dist/index.d.ts",
-	"type": "module"
+	"type": "module",
+	"packageManager": "pnpm@9.1.0+sha512.67f5879916a9293e5cf059c23853d571beaf4f753c707f40cb22bed5fb1578c6aad3b6c4107ccb3ba0b35be003eb621a16471ac836c87beb53f9d54bb4612724"
```

Comment:
I believe this is something you keep fighting to remove. Might be worth looking into a better long term solution.

Reply:
I moved it into another changelist in WebStorm now so that should keep me from committing it again

Reply:
```
pnpm dev
! The local project doesn't define a 'packageManager' field. Corepack will now add one referencing pnpm@9.1.0+sha512.67f5879916a9293e5cf059c23853d571beaf4f753c707f40cb22bed5fb1578c6aad3b6c4107ccb3ba0b35be003eb621a16471ac836c87beb53f9d54bb4612724.
! For more details about this field, consult the documentation at https://nodejs.org/api/packages.html#packagemanager
```

The hash is added by Corepack